CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Changed
 
+- `Lightnet` namespace API updates https://github.com/o1-labs/o1js/pull/1256
```

Comment:
nit: We can leave this out since we already talk about it under `CHANGED` - no need to double-specify what has changed

Reply:
the other changelog entry modified here is from much earlier!

Reply:
Ohh right - the github diff tricked me :D Then thats fine, we should just specify what actually changed!